main.tf

```
@@ -3,8 +3,8 @@ locals {
   application_name_short_full       = "${var.application_name_short}-tf"
   application_name_short_full_alnum = join("", regexall("[[:alnum:]]", local.application_name_short_full))
   environment_name_alnum            = join("", regexall("[[:alnum:]]", var.environment_name))
-  resource_group_name               = "rg${var.resource_group_number}-ss${var.subscription_number}-${local.application_name_full}-${var.environment_name}"
-  storage_account_name              = lower(substr("strg${var.resource_group_number}ss${var.subscription_number}${local.application_name_short_full_alnum}${local.environment_name_alnum}", 0, 24))
+  resource_group_name               = "rg-ss${var.subscription_number}-${local.application_name_full}-${var.environment_name}"
+  storage_account_name              = lower(substr("strgss${var.subscription_number}${local.application_name_short_full_alnum}${local.environment_name_alnum}", 0, 24))
```

Lets remove `rg` from the storage account name:
```suggestion
  storage_account_name              = lower(substr("stss${var.subscription_number}${local.application_name_short_full_alnum}${local.environment_name_alnum}", 0, 24))
```